_IBC/planq-cosmoshub.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
```

Comment:
Very nice submission. The only issue I can spot right away is the fact that you have the 'planq' name come first in these _IBC files, but in all these 3 cases, 'p' alphabetically comes after 'c', 'g', and 'o', and thus should be written second in the title and should be described by 'chain 2'.

Reply:
Oh, sorry this went totally past me.